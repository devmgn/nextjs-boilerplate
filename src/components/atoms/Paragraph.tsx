import styled from '@emotion/styled';

const Paragraph = styled.p`
  font-size: 16px;
  line-height: 24px;
`;

export default Paragraph;
