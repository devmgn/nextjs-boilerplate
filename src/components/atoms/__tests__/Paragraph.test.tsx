import { render } from '@testing-library/react';
import Paragraph from '../Paragraph';

describe('Paragraph', () => {
  test('スナップショットテスト', () => {
    const { asFragment } = render(<Paragraph>children</Paragraph>);
    expect(asFragment()).toMatchSnapshot();
  });
});
