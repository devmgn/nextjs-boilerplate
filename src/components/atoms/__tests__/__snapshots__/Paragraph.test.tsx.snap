// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph スナップショットテスト 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-size: 16px;
  line-height: 24px;
}

<p
    class="emotion-0 emotion-1"
  >
    children
  </p>
</DocumentFragment>
`;
